Week-2/assignment-1/index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This page is related to Mystay hotels">
+    <link rel="stylesheet" href="./scss/style.css">
+    <title>Mystay hotels</title>
+</head>
+
+<body>
+    <main class="parent-container">
+        <div class="layout-container">
+            <section class="hero-container">
```

Comment:
This should in header

Reply:
what I have considered is hero container is there to represent the hero section of the website so named the tag as a section and not header, from what I read Header is there to represent the brand name or branding